### PR TITLE
Insert migration state for CHW app.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@jpaidoussi ready for review.

File: go-app-chw.js [hunk 1]
```diff
@@ -673,6 +673,23 @@ go.utils = {
...
+    is_migrated_user: function (im, msisdn) {
+      var http = new JsonApi(im, {
+          headers: {
+              'Authorization': ['Token ' + im.config.identity_store.api_token]
+          }
+      });
+      return http
+        .get(im.config.identity_store.url + '/identities/search/', {
+          params: {
+            'details__addresses__msisdn': '+27001',
```
Comment (jpaidoussi) [inline go-app-chw.js]: Should this be a hardcoded msisdn or the variable?

Comment (smn) [reply go-app-chw.js]: doh!

**Comment by smn** [comment]
@jpaidoussi ready for review again

**Review by jpaidoussi — APPROVED** [review]

**Pull request merged**